spec/models/chargeback_container_project_spec.rb

```diff
@@ -45,6 +49,7 @@
   context "Daily" do
     before do
       @options[:interval] = "daily"
+      @options[:tag] = nil
```

Comment:
This keeps default `:entity_id => @project.id` from above (and no `:provider_id`).
In L251 where you set non-nil :tag, you still have :entity_id!
Can you either set all 3 in every test case, all default all to nil and set 1 in every test case?